src/connectors/TokenState.sol

```diff
@@ -5,33 +5,34 @@ pragma solidity ^0.8.17;
 import "../lib/SharedStructs.sol";
 import "forge-std/console.sol";
 import {StateIsNotEmpty} from "../errors/ConnectorErrors.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
 
 struct Transfer {
     address account;
     uint256 amount;
 }
 
-contract TokenState {
+contract TokenState is Ownable {
     uint256 public epoch;
     address[] accounts;
     mapping(address => uint256) balances; // position = 2
 
-    constructor(uint256 _epoch) {
+    constructor(uint256 _epoch) Ownable() {
```

Comment:
Who is the owner here? This contract is being deployed by the bridge contract but let's make sure that the ownership isn't on the caller.

Reply:
This contract is created(deployed?) by the Connector contract. So this Connector will be an Owner